.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: release
+    runs-on: macos-latest
```

Comment:
why on mac instead of linux?

Reply:
Literally no valid reason. I've swapped to `ubuntu-latest`